### PR TITLE
get/set feed filters via query params

### PR DESCRIPTION
**Pull request by camhashemi**
@cvializ @eishay @andrewconner 

allows users to deep link to "kifi.com/?filter=Kifi" with that team's filter pre-populated. went with whitespace-omitted team names instead of org ids so that it looks nicer.

screen shot 2016-01-19 at 3 39 29 pm

### CODE REVIEW

**Comment by andrewconner** [comment]
How about using the org's handle? It's unique.

**Comment by andrewconner** [comment]
Also, how about specifying the type of filter? `?filter=team&handle=kyfy` or something.

**Comment by andrewconner** [comment]
If you want it small, `?f=t&h=kyfy`

**Comment by camhashemi** [comment]
sounds good, though I favor using full words so that it's clear to the user what that link will do for them.

**Comment by andrewconner** [comment]
Will it be a link? I thought it was a dropdown.

**Comment by camhashemi** [comment]
the purpose of this PR is to be able to link to your filtered feed

File: kifi-backend/modules/shoebox/angular/src/feed/feedCtrl.js [hunk 1]
```diff
@@ -43,13 +45,17 @@ angular.module('kifi')
...
       { value: '', text: 'Your Stream' },
-      { value: 'own', text: 'Your Keeps' },
-      { value: 'unread', text: 'Unread'},
-      { value: 'sent', text: 'Sent'}
+      { value: 'own', text: 'Your Keeps' }
     ].concat(orgFilterOptions);
 
     // assumes this is the default setting's index ('All Keeps' 1/4/16, could be stored in prefs later on)
-    $scope.feedFilter.selected = $scope.feedFilter.options[0];
+    var allSpacesRegex = new RegExp(' ', 'g');
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/feed/feedCtrl.js]: In general you want to prefer regex literals to the regex constructor. There can be performance implications, and it's a bit cleaner and javascripty.
`var allSpacesRegex = / /g;`

Comment (camhashemi) [inline kifi-backend/modules/shoebox/angular/src/feed/feedCtrl.js]: nice, thank you.

**Comment by cvializ** [comment]
:+1: :+1:

**Comment by camhashemi** [comment]
@andrewconner now using "filter=team&handle={{handle}}" for team filters

**Pull request merged**